internal/locale/locales/en-us.yaml

```
@@ -1246,12 +1246,12 @@ arg_state_init_namespace:
   other: org/project
 arg_state_init_namespace_description:
   other: The namespace for the project that you wish to initialize
-arg_state_init_language:
-  other: Language[@version]
-arg_state_init_language_description:
-  other: The language that this project should use, and optionally the version of that language
-flag_state_init_path_description:
-  other: Where to initialize the project
+flag_state_init_language_description:
+  other: "The Language[@version] that this project should use"
+arg_state_init_path:
+  other: path
+arg_state_init_path_description:
+  other: "Where to initialize the project (Default: Current working directory)"
```

```suggestion
  other: "Where to initialize the project"
```

We don't seem to do this for other commands. Plus doing this will lead to bugs when we change the default behavior but forget the description cause it's not covered by our type system.